timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -19,11 +20,16 @@
 @Value.Immutable
 public abstract class TimeLockRuntimeConfiguration {
 
-    @Value.Default
+    @Value.Derived
     public PaxosRuntimeConfiguration paxos() {
```

Comment:
Not sure I understand the changes in this file. I htink the result here is exactly the same as before except it's just more verbose now?